seam/routes/access_codes.py

```diff
@@ -1,7 +1,8 @@
-from seam.types import AbstractAccessCodes, AbstractSeam as Seam, AccessCode
+from seam.types import AbstractSeam as Seam
+from seam.routes.types import AbstractAccessCodes, AccessCode
```

Comment:
```suggestion
from ..types import AbstractSeam as Seam
from .types import AbstractAccessCodes, AccessCode
```

Reply:
Relevant issue https://github.com/seamapi/python-next/issues/54

Reply:
Why a relative import `..types` is better than an absolute `seam.types`? As for me, absolute imports from top level looks clearer.